mds/bolt_auth.py

```diff
@@ -0,0 +1,35 @@
+class BoltClientCredentials(AuthorizationToken):
```

Comment:
This file should be removed, since it's now in `mds/api/auth.py`.

Reply:
This should be removed now.  - RMK0110

Reply:
Looks like the file is still in the branch @RMK0110 - please delete